datamodels/2.x/itop-portal-base/portal/src/Controller/ObjectController.php

```diff
@@ -1408,14 +1408,14 @@ public function GetInformationForLinkedSetAsJsonAction(Request $oRequest)
 	protected function PrepareObjectInformation(DBObject $oObject, $aAttCodes = array())
 	{
 		$sObjectClass = get_class($oObject);
-		$aObjectData = array(
-			'id'         => $oObject->GetKey(),
-			'name'       => $oObject->GetName(),
-			'attributes' => array(),
-		);
+		$aObjectData = [
+			'id'                  => $oObject->GetKey(),
+			'name'             => $oObject->GetName(),
+			'attributes'        => [],
```

Comment:
Don't know why these were added, before looked better:
```suggestion
			'id'         => $oObject->GetKey(),
			'name'       => $oObject->GetName(),
			'attributes' => [],
```